userspace/common/strl.h

```diff
@@ -45,3 +45,35 @@ static inline size_t strlcpy(char *dst, const char *src, size_t size) {
     return srcsize;
 }
 #endif
+
+/*!
+  \brief Append the NUL-terminated string src to the end of dst. It will append at most size − strlen(dst) − 1 bytes, NUL-terminating the result. 
+
+  \return The initial length of dst plus the length of src
+*/
+
+#ifndef HAVE_STRLCAT
+static inline size_t strlcat(char *dst, const char *src, size_t size) {
```

Comment:
do you think we could add a bunch of tests for this? Just to be completely sure it behaves like we want

Reply:
I agree with @Andreagit97 , even if i think this function is just taken from some source code (like bsd one?)

Reply:
makes sense, I actually had a simple test used for development :)